{
  "comments": [
    {
      "key": {
        "uuid": "ae27f2f6_01ca3943",
        "filename": "base/include/android-base/result.h",
        "patchSetId": 1
      },
      "lineNbr": 70,
      "author": {
        "id": 1415877
      },
      "writtenOn": "2019-06-13T11:18:41Z",
      "side": 1,
      "message": "Just like the previous (abandoned) change, this shows somewhat different behavior, that is, the ResultError#code() is not transferred from output to resulting ResultError.\n\nHow about separating error propagation from message formatting, in this example?\n\n    return Error(\"CalculateResult failed\") \u003c\u003c output.error();",
      "range": {
        "startLine": 70,
        "startChar": 0,
        "endLine": 70,
        "endChar": 75
      },
      "revId": "b1788fbdbd8f38c195679c37c7117e0b7aff762b",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "70d9955b_fe4dabb2",
        "filename": "base/include/android-base/result.h",
        "patchSetId": 1
      },
      "lineNbr": 70,
      "author": {
        "id": 1132673
      },
      "writtenOn": "2019-06-13T11:37:04Z",
      "side": 1,
      "message": "Done",
      "parentUuid": "ae27f2f6_01ca3943",
      "range": {
        "startLine": 70,
        "startChar": 0,
        "endLine": 70,
        "endChar": 75
      },
      "revId": "b1788fbdbd8f38c195679c37c7117e0b7aff762b",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": false
    }
  ]
}